packages/meditrak-server/src/apiV2/export/getExportPathForUser.js

```diff
@@ -3,10 +3,11 @@
  * Copyright (c) 2017 - 2021 Beyond Essential Systems Pty Ltd
  */
 import fs from 'fs';
+import { getTempDirectory } from '../../utilities';
 
-const EXPORT_BASE_PATH = 'exports';
 export const getExportPathForUser = userId => {
```

Comment:
I think this will also work:

```js
export const getExportPathForUser = userId => getTempDirectory(`exports/${userId}`);
```

... but don't bother changing it if we want to merge a fix quickly!

Reply:
Ah yes, nice!

Reply:
Tested and confirmed it worked, thanks Kostas :-)